.buildkite/packaging.pipeline.yml

```diff
@@ -172,6 +174,73 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "SNAPSHOT: {{matrix}} Linux/amd64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
+          SNAPSHOT: true
+          # packaging with `DEV=true` may cause linker issues while crosscompiling https://github.com/elastic/beats/issues/41270
+          DEV: false
+          FIPS: true
+        command: ".buildkite/scripts/packaging/package-dra.sh {{matrix}}"
+        agents:
+          provider: gcp
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+
+      - label: "SNAPSHOT: {{matrix}} Linux/arm64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_ARM_FIPS}"
+          SNAPSHOT: true
+          # packaging with `DEV=true` may cause linker issues while crosscompiling https://github.com/elastic/beats/issues/41270
+          DEV: false
+          FIPS: true
+        command: ".buildkite/scripts/packaging/package-dra.sh {{matrix}}"
+        agents:
+          provider: "aws"
+          imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+          - x-pack/agentbeat
+
+      ## Agentbeat needs more CPUs because it builds many other beats
+      - label: "SNAPSHOT: x-pack/agentbeat Linux/amd64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
```

Comment:
```suggestion
          PLATFORMS: "${PLATFORMS_AMD64_FIPS}"
```

Reply:
Fixed in e1cf0bd091e854c0a8a936fe6dfb234c0cea98ad